packages/frontend/src/hooks/cartesianChartConfig/useCartesianChartConfig.ts

```diff
@@ -24,6 +24,8 @@ import {
     sortDimensions,
 } from './utils';
 
+export const EMPTY_X_AXIS = 'empty_x_axis';
```

Comment:
can it just be null or undefined?

Reply:
I mentioned this on the description of this ticket: 

> I tried to implement this with minimal changes and side effects , so instead of setting xField to undefined, I'm giving it an special state that I can check when setting category axis and values.

Having undefined will have a big impact in the method that generates default charts.